app/views/resources/index.html.erb

```diff
@@ -14,6 +14,21 @@
         <% end %>
       </ul>
     </div>
+
+    <div class="dropdown">
+      <button class="btn btn-secondary dropdown-toggle" type="button" id="allResourcesButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        All Resources
+      </button>
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
```

Comment:
Here is example how to remove lonely `<li>` elements and avoid browser issues:
```
<div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
        <%= link_to "All Resources", resources_path, { "data-category": "*", class: "dropdown-item selected" } %>
        <% @categories.each do |category| %>
            <%= link_to category.name, resources_path(anchor: category.id), { "data-category": create_filter(category.id), class: "dropdown-item" }%>
        <% end %>
</div>
```

Reply:
done